.github/workflows/code_analysis.yml

```diff
@@ -0,0 +1,70 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
+name: CodeQL
+
+on:
+  push:
+    branches: [main, develop, releases]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [main, develop, releases]
+  schedule:
```

Comment:
Why we would like this cron schedule?

Reply:
See

https://docs.github.com/en/code-security/code-scanning/automatically-scanning-your-code-for-vulnerabilities-and-errors/configuring-code-scanning#example

Reply:
For example, you can automatically run on a pull request (PR) using GitHub Actions to make sure that changes to the codebase do not introduce new vulnerabilities. This is excellent! Not only that, new vulnerabilities are discovered regularly and having a scheduled workflow can help find vulnerabilities that weren’t previously being searched for. Both scenarios provide actionable feedback and help the team keep their application code secure.